front/src/components/Mockups.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from "react";
+import "../App.css";
+import axios from 'axios';
+import {Container} from 'reactstrap';
+
+class Mockups extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            mockups: []
+        }
+    }
+
+    componentDidMount() {
+        axios.get('http://localhost:8080/mockups/all')
+            .then((response) => {
+                const data = response.data;
+                this.setState({mockups: data});
+            })
+            .catch(() => {
+                console.log("Error");
```

Comment:
Error messages could be more specific

Reply:
And even better would be to create some popup, to let user know that sth is wrong